x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/add_source_logic.ts

```diff
@@ -430,9 +419,7 @@ export const AddSourceLogic = kea<MakeLogicType<AddSourceValues, AddSourceAction
         ? `/internal/workplace_search/org/sources/${sourceId}/reauth_prepare`
         : `/internal/workplace_search/account/sources/${sourceId}/reauth_prepare`;
 
-      const query = {
-        kibana_host: kibanaHost,
-      } as HttpFetchQuery;
+      const query = {} as HttpFetchQuery;
 
       try {
         const response = await HttpLogic.values.http.get<SourceConnectData>(route, { query });
```

Comment:
I believe we can omit the `query` here

```suggestion
        const response = await HttpLogic.values.http.get<SourceConnectData>(route);
```